keras_nlp/benchmarks/README.md

```diff
@@ -0,0 +1,68 @@
+# KerasNLP Benchmarks
+
+This directory houses a collection of scripts for benchmarking APIs and utility
+functions which KerasNLP provides.
+
+## Text Generation
+For benchmarking text generation functions, the following command can be run
+from the root of the repository:
+
+```sh
+python3 ./keras_nlp/benchmarks/text_generation.py
+```
+
+On running this script on Google Colab, the following results were obtained:
```

Comment:
These metrics are subject to change, I would prefer delete them. @mattdangerw what's your opinion on this?

Reply:
Waiting for Matt's opinion on this

Reply:
Oh sorry missed this. I think it helps to have the numbers to track!

In a lot of ways this benchmark seems like a pilot. If we notice these numbers get stale, we can always delete them later. I don't see a cost to adding.

Reply:
Cool! I'm thinking of making a table instead of pasting the raw output. And, will mention the GPU version, etc.